### PR TITLE
Show Total Cost in UI

### PR DESCRIPTION
**Pull request by jiaweiteo**

### CODE REVIEW

File: src/main/java/seedu/address/model/ModelManager.java [hunk 1]
```diff
@@ -234,6 +234,7 @@ public void setTravelPlanObject(TravelPlanObject target, TravelPlanObject edited
         requireAllNonNull(target, editedTravelPlanObject);
         assert directory instanceof TravelPlan : "Directory must be set to a TravelPlan to call setTravelPlanObject.";
         TravelPlan tp = (TravelPlan) directory;
+        Directory d = tp;
```
Comment (lyeyixian) [inline src/main/java/seedu/address/model/ModelManager.java:237]: What does this line suppose to do?

File: src/main/java/seedu/address/ui/TravelPlanPanel.java [hunk 2]
```diff
@@ -6,23 +6,30 @@
...
  * Panel containing basic information of a Travel Plan.
  */
 public class TravelPlanPanel extends UiPart<Region> {
     private static final String TravelPlanFXML = "TravelPlanPanel.fxml";
+    private static final String TOTAL_COST = "Total Cost: ";
     private final Logger logger = LogsCenter.getLogger(TravelPlanPanel.class);
 
     private Directory directory;
 
+    private Logic logic;
```
Comment (lyeyixian) [inline src/main/java/seedu/address/ui/TravelPlanPanel.java:25]: Why do you need logic here tho? Doesn't seem to be used anywhere in the changes in this class?

File: src/main/java/seedu/address/ui/TravelPlanPanel.java [hunk 3]
```diff
@@ -39,11 +46,16 @@ public TravelPlanPanel(Directory directory) {
     public void update() {
         if (directory instanceof TravelPlan) {
             TravelPlan travelPlan = (TravelPlan) directory;
+            String cost = travelPlan.getTotalCost();
```
Comment (lyeyixian) [inline src/main/java/seedu/address/ui/TravelPlanPanel.java:49]: Now no need do typecasting already. You can add this getTotalCost() method into Directory class.

Comment (lyeyixian) [reply src/main/java/seedu/address/ui/TravelPlanPanel.java:49]: But still need to check isTravelPlan() to setText lah

File: src/main/java/seedu/address/ui/TravelPlanObjectListPanel.java [hunk 4]
```diff
@@ -85,6 +87,7 @@ public void setAllTabsVisible() {
         @Override
         protected void updateItem(Activity activity, boolean empty) {
             super.updateItem(activity, empty);
+            travelPlanPanel.update();
```
Comment (lyeyixian) [inline src/main/java/seedu/address/ui/TravelPlanObjectListPanel.java:90]: Is calling the update() here suitable?

File: src/main/java/seedu/address/ui/MainWindow.java [hunk 5]
```diff
@@ -159,6 +159,15 @@ private void handleDirectoryChange(Directory directory) {
         }
     }
 
+    private void handleCostChange(Directory directory) {
```
Comment (lyeyixian) [inline src/main/java/seedu/address/ui/MainWindow.java:162]: I think the travelPlanPanel.update() can be called here instead? So, no need to pass into TravelPlanObjectListPanel

File: src/main/java/seedu/address/model/ModelManager.java (hunk 1 above)
Comment (jiaweiteo) [reply src/main/java/seedu/address/model/ModelManager.java:237]: deleted

File: src/main/java/seedu/address/ui/TravelPlanPanel.java (hunk 2 above)
Comment (jiaweiteo) [reply src/main/java/seedu/address/ui/TravelPlanPanel.java:25]: deleted

File: src/main/java/seedu/address/ui/TravelPlanPanel.java (hunk 3 above)
Comment (jiaweiteo) [reply src/main/java/seedu/address/ui/TravelPlanPanel.java:49]: ok solved

File: src/main/java/seedu/address/ui/TravelPlanObjectListPanel.java (hunk 4 above)
Comment (jiaweiteo) [reply src/main/java/seedu/address/ui/TravelPlanObjectListPanel.java:90]: No choice...

**Pull request closed without merging**